src/Zicht/Bundle/SolrBundle/Manager/Doctrine/Subscriber.php

```diff
@@ -20,7 +21,7 @@ class Subscriber implements EventSubscriber
      * @var ContainerInterface
      */
     protected $container;
```

Comment:
This `$container` property has been made `protected` in v4, but in v3 it is `private`. I kept it `protected` (unchanged) during the merge as you can see. Now I get this error in a project where I'm upgrading it from v3.2.x to this merge branch:

> PHP Fatal error:  Access level to Zicht\Bundle\XxxyyyzzzSiteBundle\Solr\SiteSubscriber::$container must be protected (as in class Zicht\Bundle\SolrBundle\Manager\Doctrine\Subscriber) or weaker in src/Zicht/Bundle/XxxyyyzzzSiteBundle/Solr/SiteSubscriber.php on line 15

But you could expect to have to do some fixing when upgrading to a next major version...